File: PyramidPatternProblem/JAVA/PyramidPattern1.java
//Solid rectangular star pattern:
//Number of rows: 3
//Number of columns: 5

//* * * * *
//* * * * *
//* * * * *
//Program to print solid rectangular star pattern
import java.util.*;

public class Main 

{ 

static void solid_rectangle(int n, int m)

{

int i, j;

for (i = 1; i <= n; i++)

{

for (j = 1; j <= m; j++)

{

System.out.print(“*”);

}

System.out.print(“n”);

}



}

public static void main (String args[])

{ 

int rows, columns;

Scanner sc = new Scanner(System.in);

System.out.print(“nEnter the number of rows : “);

rows = sc.nextInt();

System.out.print(“nEnter the number of columns : “);

columns = sc.nextInt();

System.out.print(“n”);

solid_rectangle(rows, columns);

} 

}

File: PatternProblems/PyramidPatternProblem/PYTHON/pyramidpattern1.py
# Python program to print solid rectangle star pattern


#Number of rows: 3
#Number of columns: 5

#* * * * *
#* * * * *
#* * * * *
#Program to print solid rectangular star pattern
# Function to print solid rectangle

def solid_rectangle(n, m):

for i in range(1, n+1):

for j in range(1, m+1):

print(“*”, end = “”)

print()

rows = int(input(“Enter the number of rows : “))

columns = int(input(“Enter the number of columns : “))

solid_rectangle(rows, columns)